backend/Origam.Rule/RuleEngine.cs

```diff
@@ -3300,26 +3300,7 @@ private bool ProcessRulesInternalFinish(ArrayList rules, IDataDocument data, Dat
 					{
 						throw new NullReferenceException("Rule has IsPathRelative set but no XmlDataDocument has been provided. Cannot evaluate rule.");
 					}
-
```

Comment:
There are some really important notes here. It brings me to an idea that since we dumped the `XmlDataDocument` maybe the no-go code would actually start working.

Reply:
Please do not remove these notes just yet.